src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuilding.java

```diff
@@ -1376,14 +1376,14 @@ private ItemStack forceItemStackToProvider(@NotNull final ICapabilityProvider pr
      */
     public void setMirror()
     {
-        this.isMirrored = !isMirrored;
+        this.isBuildingMirrored = !isBuildingMirrored;
```

Comment:
shouldn't this method be `invertMirror` or something? or make it `setMirror(final boolean isMirrored)` ?

Reply:
Good idea =)